src/main/kotlin/day17/dto/photo/PhotoDto.kt

```
@@ -0,0 +1,55 @@
+    val id: Int?,
+    val type: String? = null,
+    val title: String? = null,
+    val content: String? = null,
+    val url: String? = null,
+    val caption: String? = null,
+    @SerialName("created_at") val createdAt: String?,
+) {
+    companion object {
+        fun checkType(type: PhotoType) {
+
+        }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)